input/fsh/au-erequesting-path-request.fsh

```diff
@@ -0,0 +1,23 @@
+Profile: AUeRequestingPathologyServiceRequest
+Parent: AUeRequestingServiceRequest
+Id: au-erequesting-path-request
```

Comment:
the id if following the conventions in base and core would be either be 'au-erequesting-servicerequest-path' or 'au-erequesting-pathrequest'. Likely the former one is most correct since this is based on 'au-erequesting-servicerequest'

Reply:
Changed to 'au-erequesting-pathrequest'

Reply:
Why the later (-pathrequest) over the former that is used when we derive off a profile in the ig (-servicerequest-path)?

Reply:
Bc I for some reason I decided that it follows the conventions. Changed to "au-erequesting-servicerequest-path"